modules/administration_manual/pages/enterprise/classification_and_policy_enforcement.adoc

```diff
@@ -0,0 +1,329 @@
+= Document Classification and Policy Enforcement
+
+When dealing with large amounts of data in an enterprise, it is essential to have mechanisms in place that allow you to stay in control of data flows.
+To implement such mechanisms the first step to take is to define guidelines that describe how contents of different security levels have to be treated.
+
+Depending on the industry such information security guidelines can originate from regulatory requirements, from recommendations of industry associations or they can be self-imposed if there's no external factor but internal risk management requirements that demand special treatment for specific information.
+
+The leading information security standard https://www.iso.org/isoiec-27001-information-security.html[ISO 27001] defines guidelines for managing information security which can be certified:
+
+. Information should enter an asset inventory (A.8.1.1)
+. Information should be classified (A.8.2.1)
+. Information should be labeled (A.8.2.2)
+. Information should be handled in a secure way (A.8.2.3)
+
```

Comment:
We may add a note regarding ``GDPR General Data Protection Regulation`` to be found at ``https://gdpr-info.eu``

Reply:
At this stage, I don't feel that it's necessary, as it may result in information overload.

Reply:
I have a different feeling, especially for enterprise environments ...

https://www.certificationeurope.com/insights/gdpr-iso-27001-mapping-tool-now-available/
```
As the leading international standard and certification for information security, 
ISO 27001 covers 75-80% of GDPR. 
This makes it the ideal choice of a framework to support GDPR compliance.
```
https://www.certificationeurope.com/app/uploads/2018/05/GDPR-ISO-27001-Mapping-Guide.pdf
This might be useful as it maps the GDPR Articles to ISO Controls

It is not necessary to write a lot about, but those infos/links are very useful and supportive, eg the mapping guide above matches your A.8.x Controls to the relevant GDPR Articles. 

It might be enough to write:

As the leading international standard and certification for information security,  ISO 27001 covers 75-80% of GDPR. This makes it the ideal choice of a framework to support GDPR compliance.

Please see the following link as an example to match the mentioned A.8.x Controls to the relevant General Data Protection Regulation (GDPR) Articles (pdf link from above).

Reply:
Wow, this is great information and a perfect use case I didn't yet think of. Thanks a lot @mmattel. I would like to add some points these regards. The suggested sentences fit well + link to source. Also I would like to add some example categories (in addition to "Public, Internal, Confidential...") in regards of GDPR, like "No PID (Personally Identifiable Information)", "PID" and "Special PID", to shed some light on this use case and show for how many areas this new feature is beneficial.

Reply:
In my very personal opinion, these are the things which differenciate us and makes the intention why these things are implemented and good for more visible and tangible.
As natural it may sound, but this is a sales story cast into documentation...

Reply:
Okay, I'm convinced. I'll get the changes made. Thanks, @mmattel (_as always_), @pmaier1.

Reply:
The changes have been made. You can find them at https://github.com/owncloud/docs/pull/160/files#diff-89f11ff444d5711663340e056e8cdfc0R16 and 
https://github.com/owncloud/docs/pull/160/files#diff-89f11ff444d5711663340e056e8cdfc0R24.

Reply:
The latest copy of master's been rebased on this branch, so if you're both happy with the additions, then let's merge it. Otherwise, let me know what further additions are required, and I'll get those in.